README.md

```diff
@@ -1,59 +1,117 @@
 
 ## Prerequisites
     Node Enviroment 
     Git 
     Firebase Database Account
     Postman
-    Google Chrome
+    Google Chrome 
 
 
-## Deployment
+# Deployment
  PostIt is hosted on heroku and can be accessed via this link
-    https://post-it-app35.herokuapp.com/
+        https://post-it-app35.herokuapp.com/
 
 
+# Installation
+  Kindly follow the steps below to setup a local development environment.
+  1. Clone this repository from a terminal git clone https://github.com/Charpell/post-it-app.git.
+  2. Move into the project directory
+  3. Install project dependencies npm install
+  4. Open 2 git bash terminal, one will run the server and the other runs webpack
+  6. Run 'npm start'
+  7. Run 'webpack -w'
 
-### Service API's:
-  - <a href="https://post-it-app35.herokuapp.com/user/signup">Sign Up</a>
-  - <a href="https://post-it-app35.herokuapp.com/user/signin">log-in</a>
-  - <a href="https://post-it-app35.herokuapp.com/group">Create group</a>
-  - <a href="https://post-it-app35.herokuapp.com/group/:groupID/:user">Add a user to your group</a>
-   
-   
-   
-   
 
+# To Run Test
+  The mocha framework and supertest module are used for testing our routes.   
+  To test, Run "mocha"
+ 
+### To test the API with postman:
+- run ```npm start```
+- then visit ```http://localhost:3000```, you should see 'PostIt API running...' 
+
+It is also hosted on heroku at <a href="https://post-it-app35.herokuapp.com//" target="_blank">PostIt API</a>.
+
+The API contains different endpoints with their respective payloads as stated below:
 
+| Endpoints                    | Functions                                                               | Payloads                 | Request Methods |
+|------------------------------|-------------------------------------------------------------------------|--------------------------|-----------------|
+|1 /user/signup             | It allows users to register                                             | username, email and password    | POST            |
+|2 /user/google             | It allows users to register                                             | google auth    | POST            |
+|3 /user/signin             | It gives users access to login                                          | email and password    | POST            |
+|4 /group                   | It allows users to create group for notifications                       | groupName and userName | POST            |
+|5 /group/:groupName/:user     | It allows users to add another user to a created group of id groupID    | username                 | POST            |
+| /groups/:groupName/:messages
+/:emails/:numbers/:allUsers
+/:notification/:priority/group
+/:groupID/message  | It allows users to post message to a created group of id groupID        | message and postedBy     | POST            |
+|6 /group/:groupID/messages | It allows users to retrieve messages from a created group of id groupID | No payload               | GET             |
+
+
```

Comment:
Some endpoints have no number or description. Also I see you used groupName and groupID and I'm a bit confused on why/when you use them.

Suggestion: Use `POST /groups` rather than `POST /group` to create new groups.